src/Main.java

```diff
@@ -1,5 +1,99 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Привет, Мир!");
+        System.out.println("Task 1");
+        int age = 18;
+        if (age >= 18) {
+            System.out.println("Если возраст человека равен 18, то он совершеннолетний");
+        }
+        int age2 = 17;
```

Comment:
зачем заводить новую переменную с плохим названием (цифры на конце названия - это всегда плохо), если можно было просто переиспользовать уже созданную ранее переменную
```suggestion
        age = 17;
```